pages/samvera/developer_community/release_process.md

```diff
@@ -0,0 +1,105 @@
+
+For each cycle the Release Testing facilitator will:
+
+- Communicate with testers and demo site committers about timing of upcoming release.
+- Ensure the Managers Guide, UI Interactions, Test Tracking Spreadsheet and demo site are up to date for the process of testing and documentation.
+- Recruit help with testing, bug reporting and documentation.
+- Ensure testing and documentation is complete.
+
+
+This testing process complements test coverage and regression testing in the code but does not replace it. This testing is meant to facilitate browser, platform, feature, (basic) usability and completion, and accessibility.
+
+### Major release process:
+**Beta Release:** All major features are complete. 2-3 weeks
+
+- Beta release on demo site
+
+- Testing for user acceptance and general readiness for systematic testing (no major issues identified that would impede testing).
+
+- Beta release is announced
+
+- First draft of documentation is done
+
+- First round of testing and bug reporting begins
+
+- Institutions have a chance to report bugs or ask for a PR to be considered for inclusion in release
+
+**Release Candidate:** Feature Freeze and bug fixes only. 2-3 weeks (may be unrealistic time frame?)
+
+- RC goes to demo site
+
+- RC is announced
+
+- Finalize feature documentation by end of phase
+
+- Finalize testing document by end of phase
+
+- Distributed testing of all features, supported browsers, accessibility testing.
+
+- Report bugs
+
+- Confirm bugs (PO?) and determine priority (need to establish process for this)
+
+- Fix all priority bugs
+
+- Repeat above with additional versions of RCx until release is confirmed stable.
+
+**Final Release:**
+
+- Once Final Release is ready, at least 1 in-production institutions should upgrade before wide release (This may not be doable, but would be good if there are institutions willing to do this, could rotate and plan ahead to help curb)
+
+- Final Release is announced widely
+
+### Minor release process:
+
+- Follows the Major release process on a much more compressed timeline.
+
+- Upgrade demo site, retaining data. In the occasion that data needs to be wiped for whatever reason, this is fine. Mostly we want to test that data and users migrate smoothly.  
+
+### Hyrax Shared Testing Tools:
+
+**Demo site:** shared testing and demo instance of Hyrax. Vanilla app with all "bonus" features configured for testing.
+
+  - https://nurax.curationexperts.com hosted by DCE
+  - Github repo: https://github.com/curationexperts/nurax/
+  - Maintained and updated by committers in DCE and Samvera Community
+
+**Hyrax feature guide:** describes features, set up and concepts for Hyrax repositories
+
+ - [http://samvera.github.io/](http://samvera.github.io/)
```

Comment:
This link goes to the main site. Perhaps it would be helpful to include the name of the section heading to open... it's not called "feature guide" on the documentation site.